optashift-employee-rostering-gwtui/src/main/java/org/optaplanner/openshift/employeerostering/gwtui/client/beta/java/model/Blob.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.openshift.employeerostering.gwtui.client.beta.java.model;
+
+public interface Blob<T> {
+
+    T getPositionInScaleUnits();
+
+    void setPositionInScaleUnits(final T position);
+
+    //FIXME: Change it to getSizeInScaleUnits?
+    Long getSizeInGridPixels();
+
+    void setSizeInGridPixels(final Long sizeInGridPixels);
```

Comment:
Yeah, this a complex decision. Some things are designed to have a start point and an end point, but not a size (like Shifts). For example: if you have a time scale like this, in what unit is the size represented? Minutes, hours, seconds? So I thought it would be better to leave it in grid pixels, because is a common unit for every blob implementation.